### PR TITLE
feat: added container to the API

### LINKED ISSUE #23: [Feature]: add container to the exported API
```js
import Button from 'Button.svelte';
import {render} from 'svelte-testing-library'

const { container } = render(Button);

//..

expect(container.firstChild).toMatchSnapshot();
```

This will allow to create snapshot tests in a easier way. It could be a nice addition as other `testing-library` libraries export it. I know it's possible to use `document.body` but it's not intuitive and not easy for new comers

### PR DESCRIPTION
**Pull request by ematipico**
This PR closes #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ import App from '../src/App.svelte'
 import {render} from 'svelte-testing-library'
 describe('App', () => {
   test('should render greeting', () => {
-    const {getByText} = render(App, {props: {name: 'world'}})
+    const {getByText, container} = render(App, {props: {name: 'world'}})
```
Comment (benmonro) [inline README.md]: can you remove this example from here and move it into its own section for containers in the docs website?  https://testing-library.com/docs/svelte-testing-library/intro

File: README.md [hunk 2]
```diff
@@ -129,6 +129,7 @@ describe('App', () => {
     const button = await waitForElement(() => getByText('Button Clicked'))
 
     expect(button).toBeInTheDocument()
+    expect(container.firstChild).toMatchSnapshot()
```
Comment (benmonro) [inline README.md]: same as above

File: README.md (hunk 1 above)
Comment (benmonro) [reply README.md]:
```suggestion
    const {getByText} = render(App, {props: {name: 'world'}})
```

File: README.md (hunk 2 above)
Comment (benmonro) [reply README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (ematipico) [reply README.md]: Will do!

**Pull request merged**